### PR TITLE
Show angry smile after foe name

### LINKED ISSUE #887: Show indication when joining game containing a foe

### PR DESCRIPTION
**Pull request by 1-alex98**
Fixes #887

### CODE REVIEW

File: src/test/java/com/faforever/client/game/PlayerCardTooltipControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    PlayerBuilder playerBuilder = PlayerBuilder.create("user")
+        .defaultValues()
+        .socialStatus(SocialStatus.FOE);
+    Player player = playerBuilder.get();
+    instance.setPlayer(player, 1000);
+
+    assertThat(instance.foeIconText.isVisible(), is(true));
+    assertThat(instance.playerInfo.getText(), is("user(1000)"));
+  }
+}
```
Comment (Geosearchef) [inline src/test/java/com/faforever/client/game/PlayerCardTooltipControllerTest.java:75]: pls add a test for non foes.

**Review by Geosearchef — DISMISSED** [review]

File: src/test/java/com/faforever/client/game/PlayerCardTooltipControllerTest.java (hunk 1 above)
Comment (1-alex98) [reply src/test/java/com/faforever/client/game/PlayerCardTooltipControllerTest.java:75]: We happy a added tests XD

**Comment by micheljung** [comment]
Could you post a screenshot of the result? :)

**Comment by 1-alex98** [comment]
image

**Comment by 1-alex98** [comment]
image

**Comment by 1-alex98** [comment]
@Geosearchef would u review again?

**Review by Geosearchef — APPROVED** [review]

**Review by johnhce — APPROVED** [review]

**Pull request merged**